### PR TITLE
Rework subscription descriptions

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary

This pull request reworks subscription descriptions, and related files, to conform to the [writing style](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md).

## Links

N/A

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/Plans/FlexiblePlan.yaml [hunk 1]
```diff
@@ -3,6 +3,6 @@ allOf:
   - $ref: ./Plan.yaml
 properties:
   id:
-    description: ID of a plan you wish to modify.
+    description: ID of a plan you want to modify.
```
Comment (Boydbueno) [inline openapi/components/schemas/Plans/FlexiblePlan.yaml:6]: Suggestion: I think we can leave the second part of this sentence, for consistency. 
```suggestion
    description: ID of the plan.
```

**Review by nanobash — DISMISSED** [review]

File: openapi/components/schemas/Subscription/SubscriptionCancellation.yaml [hunk 2]
```diff
@@ -62,14 +62,19 @@ properties:
...
+      draft: Creates a daft cancellation so that the cancellation and charge can be previewed.
+      confirmed: |-
+        Confirms a subscription cancellation.
+        Sets the subscription to cancel when the `churnTime` is reached.
       completed: |-
-        Read-only status set by the system when the `churnTime` is reached.
-        The cancellation may not be changed or deleted when the status is "completed".
-      revoked: Cancellation is revoked.
+        Marks a subscription cancellation as completed.
+        This is a read-only status that set when the `churnTime` is reached.
```
Comment (Boydbueno) [inline openapi/components/schemas/Subscription/SubscriptionCancellation.yaml:71]:
```suggestion
        This is a read-only status that is set when the `churnTime` is reached.
```

**Review by Boydbueno — DISMISSED** [review]

File: openapi/components/schemas/Common/CommonBankAccount.yaml [hunk 3]
```diff
@@ -43,7 +43,12 @@ properties:
     allOf:
       - $ref: ../Contact/ContactObject.yaml
   fingerprint:
-    description: Unique value which identifies the bank account. This value contains alphanumeric characters.
+    description: |-
+      Unique value which identifies the bank account.
+      A device fingerprint is a unique token that is used to identify the customer.
+      The device fingerprint is generated based on device attributes,
+      such as: hardware, software, IP address, language, browser, and more.
```
Comment (ashkarpetin) [inline openapi/components/schemas/Common/CommonBankAccount.yaml:50]: I don't think this is correct, this is not a device fingerprint. Bank account instrument fingerprint is generated from BBAN and routing number, or IBAN.

File: openapi/components/schemas/Common/CommonKhelocardCard.yaml [hunk 4]
```diff
@@ -12,7 +12,12 @@ properties:
     enum:
       - Khelocard
   fingerprint:
-    description: Unique value to identify the payment instrument regardless of variable values. It contains alphanumeric values.
+    description: |-
+      Unique value to identify the payment instrument regardless of variable values.
+      A device fingerprint is a unique token that is used to identify the customer.
+      The device fingerprint is generated based on device attributes,
+      such as: hardware, software, IP address, language, browser, and more.
```
Comment (ashkarpetin) [inline openapi/components/schemas/Common/CommonKhelocardCard.yaml:19]: I don't think this is correct, this is not a device fingerprint. Khelocard fingerprint is generated from number, cvv and expiration.

File: openapi/components/schemas/Common/CommonPaymentCard.yaml [hunk 5]
```diff
@@ -31,6 +31,9 @@ properties:
   fingerprint:
     description: |-
       Unique value that is used to identify the payment instrument regardless of variable values.
+      A device fingerprint is a unique token that is used to identify the user.
+      The device fingerprint is generated based on device attributes,
+      such as: hardware, software, IP address, language, browser, and more.
```
Comment (ashkarpetin) [inline openapi/components/schemas/Common/CommonPaymentCard.yaml:36]: I don't think this is correct, this is not a device fingerprint. Card fingerprint is generated from first6 and last6.

File: openapi/components/schemas/Common/CommonBankAccount.yaml (hunk 3 above)
Comment (noon-dawg) [reply openapi/components/schemas/Common/CommonBankAccount.yaml:50]: Excellent input! Thank you 🙏

File: openapi/components/schemas/Common/CommonBankAccount.yaml [hunk 6]
```diff
@@ -43,7 +43,13 @@ properties:
...
       - $ref: ../Contact/ContactObject.yaml
   fingerprint:
-    description: Unique value which identifies the bank account. This value contains alphanumeric characters.
+    description: |-
+      Unique value which identifies the bank account.
+      This value contains alphanumeric characters.
+      Depending on the type of bank account number,
+      a bank account fingerprint is generated using one of the following:
+      - BBAN: `last4` and `routingNumber` values.
+      - IBAN: First 8 digits of the IBAN and the `last4` value.
```
Comment (ashkarpetin) [inline openapi/components/schemas/Common/CommonBankAccount.yaml:52]:
```suggestion
      - IBAN: First 8 characters of the IBAN and the `last4` value.
```
IBAN contains alphanumeric characters, i.e. DE75512108001245126199

**Review by ninjasimon — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Review by nanobash — APPROVED** [review]

**Pull request merged**